recipes/halide/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "2016_04_27" %}
+
+package:
+  name: halide
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/Halide/halide.git
+  git_tag: release_{{ version }}
+
+build:
+  number: 0
+  skip: True    # [win]
+
+requirements:
+ build:
+        - gcc     # [osx]
+        - llvm
+        - cmake
+    run:
+        - libgcc  # [osx]
+
+  run:
+    - python
+
+#test:
+#  imports:
+#    - example
+#  commands:
+#    - python -c "import example;"
+
+about:
+  home: https://github.com/halide/Halide
+  license: BSD
```

Comment:
@arokem can you check which version of the BSD license? `BSD 2-Clause` or `BSD 3-Clause`?